packages/x-components/src/views/home/Home.vue

```diff
@@ -76,10 +76,15 @@
               <div class="x-input-group x-input-group--card">
                 <SearchInput
                   aria-label="Search for products"
-                  placeholder="Search"
                   :instant="controls.searchInput.instant"
                   :instant-debounce-in-ms="controls.searchInput.instantDebounceInMs"
                 />
+                <SearchInputPlaceholder
+                  :animatedPlaceholders="placeholders"
```

Comment:
Not super happy about the name of this prop.

Reply:
Haha, I'm not convinced either by it but I can't think of better options. Maybe one of: `contentToAnimate`, `textsToAnimate`, `placeholdersToAnimate`,  or just `items`?